### ISSUE TITLE
installing glumpy using pip

### ISSUE THREAD

**Issue by gankrish** [glumpy/glumpy#24]
-----------------------------------------------------------------------------------------------------------------------
sudo pip install glumpy
Downloading/unpacking glumpy
  Downloading glumpy-1.0.3.tar.gz (485Kb): 485Kb downloaded
  Running setup.py egg_info for package glumpy
    /usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'platform'
      warnings.warn(msg)
    
    package init file 'glumpy/ext/sdf/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/transforms/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/collections/__init__.py' not found (or not a regular file)
    warning: no files found matching 'TODO.txt'
    warning: no files found matching 'README.rst'
    warning: no files found matching 'CHANGELOG.txt'
    warning: no files found matching 'MANIFEST.IN'
Requirement already satisfied (use --upgrade to upgrade): numpy in /usr/lib/python2.7/dist-packages (from glumpy)
Requirement already satisfied (use --upgrade to upgrade): triangle in /usr/local/lib/python2.7/dist-packages/triangle-2013.04.05-py2.7-linux-x86_64.egg (from glumpy)
Requirement already satisfied (use --upgrade to upgrade): cython in /usr/local/lib/python2.7/dist-packages (from glumpy)
Installing collected packages: glumpy
  Running setup.py install for glumpy
    /usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'platform'
      warnings.warn(msg)
    package init file 'glumpy/ext/sdf/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/transforms/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/collections/__init__.py' not found (or not a regular file)
    skipping 'glumpy/ext/sdf/_sdf.c' Cython extension (up-to-date)
    building 'glumpy.ext.sdf' extension
    gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/include/python2.7 -c glumpy/ext/sdf/_sdf.c -o build/temp.linux-x86_64-2.7/glumpy/ext/sdf/_sdf.o
    /usr/lib/python2.7/dist-packages/numpy/core/include/numpy/__ufunc_api.h:226:1: warning: ‘_import_umath’ defined but not used [-Wunused-function]
    gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/include/python2.7 -c glumpy/ext/sdf/sdf.c -o build/temp.linux-x86_64-2.7/glumpy/ext/sdf/sdf.o
    gcc -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-Bsymbolic-functions -Wl,-z,relro build/temp.linux-x86_64-2.7/glumpy/ext/sdf/_sdf.o build/temp.linux-x86_64-2.7/glumpy/ext/sdf/sdf.o -o build/lib.linux-x86_64-2.7/glumpy/ext/sdf.so
    
    warning: no files found matching 'TODO.txt'
    warning: no files found matching 'README.rst'
    warning: no files found matching 'CHANGELOG.txt'
    warning: no files found matching 'MANIFEST.IN'
Successfully installed glumpy
Cleaning up...

-------------------------------------------------------------------------------------------------------------------------
I start the python console and I am not able to import glumpy. During the installation (see above)the messages
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'platform'
      warnings.warn(msg)
    package init file 'glumpy/ext/sdf/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/transforms/__init__.py' not found (or not a regular file)
    package init file 'glumpy/library/collections/__init__.py' not found (or not a regular file)

I saw a similar post and you seem to have closed it. So I raised a new issue.

**Comment by rougier** [comment]
I think your problem is new.

The warning install seems to be related to the setuptools not being present on your system (see http://stackoverflow.com/questions/8295644/pypi-userwarning-unknown-distribution-option-install-requires)

Could you try:
```
$ pip uninstall glumpy
$ pip install setuptools
$ pip install glumpy
```
And if this does not work, can you post the Traceback from
```
>>> import glumpy
```

**Comment by gankrish** [comment]
I installed glumpy after upgrading setuptools and I was able to import glumpy without any error. However, there is another problem. I will post it here but if you like I can start a new issue.
I am using glumpy as part of the toast++ library for optical tomography. One of the tutorial scripts has the following line of code.

from glumpy import figure
which gives the error

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name figure
